src/RaisedButton/RaisedButton.spec.js

```diff
@@ -91,4 +91,16 @@ describe('<RaisedButton />', () => {
       );
     });
   });
+
+  it('inherits fontSize from theme', () => {
+    const wrapper = shallowWithContext(
+      <RaisedButton label="test" />
+    );
+
+    assert.strictEqual(wrapper.contains('test'), true);
```

Comment:
What about this? `wrapper.text()` doesn't work since there are non-primitive components in the way (just returns `<Paper />`. The alternative is something like `wrapper.find('EnhancedButton').childAt(0).text()`.